ilastik/workflows/tracking/structured/structuredTrackingWorkflow.py

```diff
@@ -109,17 +113,26 @@ def __init__( self, shell, headless, workflow_cmdline_args, project_creation_arg
         self.default_tracking_export_filename = '{dataset_dir}/{nickname}-tracking_exported_data.csv'
         self.dataExportTrackingApplet = TrackingBaseDataExportApplet(self, "Tracking Result Export",default_export_filename=self.default_tracking_export_filename)
         opDataExportTracking = self.dataExportTrackingApplet.topLevelOperator
-        opDataExportTracking.SelectionNames.setValue( ['Tracking Result', 'Merger Result', 'Object Identities'] )
+        opDataExportTracking.SelectionNames.setValue( ['Tracking-Result', 'Merger-Result', 'Object-Identities'] )
         opDataExportTracking.WorkingDirectory.connect( opDataSelection.WorkingDirectory )
```

Comment:
One minor point: In other workflows, we use spaces in the export "selection names", not hyphens.  IMHO, it would be better to mimic the conventions used in our other workflows (e.g. Object Classification), unless that is problematic for some reason.